packages/core/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "pcln-design-system",
```

Comment:
Maybe we should keep the same naming for the folder as `pcln-design-system` or `design-system` for short if we will be transitioning to be serving `@pcln/design-system` in v2.

Reply:
This has to be the same name as the package that’s published. AFAIK we weren’t planning on renaming the package

Reply:
Oh, I think I misread this. Were you suggesting renaming the directory from `core` to `design-system`? My only thinking there is then you end up with a sort of redundant `design-system/packages/design-system`, but happy to name it whatever makes most sense

Reply:
Yes, but good point on the path. Following the reasoning behind using lerna, we will end up having a set of packages that would include priceline UI resources such as date-picker so maybe the root name would have to be changed to something broader than `design system` in the future. Also, I am just thinking that it would make sense to have a set of npm packages with the `@pcln` prefix following the same patterns used on other internal utils packages. `@pcln/design-system`, `@pcln/date-picker`, `@pcln/icons` would make sense to me.

Reply:
We weren't planning on using scope for the public npm packages, but could be worth discussing at the next meeting